libtelemetry/src/test/java/com/mapbox/android/telemetry/MockWebServerTest.java

```diff
@@ -14,8 +14,11 @@
 import java.util.Arrays;
 import java.util.List;
 
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSession;
+
 import okhttp3.HttpUrl;
-import okhttp3.internal.tls.SslClient;
+import okhttp3.mockwebserver.internal.tls.SslClient;
```

Comment:
Should we continue to use an internal api if future changes could cause issues similar issues? Is there any way we can get rid of this?

Reply:
Here is a bit different because MockWebServer is only used for testing. So although APIs could change it's not a problem because it's not affecting production code which is what we're exposing and what could cause issues in the future. Does that make sense?

Reply:
> Is there any way we can get rid of this?

Nope, it's actually needed for testing HTTP requests.

Reply:
Makes sense. Sounds good.